openassessment/xblock/studio_mixin.py

```diff
@@ -379,19 +347,13 @@ def _editor_assessments_order_context(self):
         # since the user last saved their ordering.
         effective_order = copy.deepcopy(DEFAULT_EDITOR_ASSESSMENTS_ORDER)
 
-        # If the problem already contains example-based assessment
-        # then allow the editor to display example-based assessments,
-        # which is not included in the default
-        enabled_assessments = [asmnt['name'] for asmnt in self.valid_assessments]
```

Comment:
Have you checked that no ORA instances in prod/Edge have AI steps defined? Just wondering how they would fail.